__tests__/app.test.js

```diff
@@ -127,29 +128,74 @@ describe("GET /api/articles", () => {
     });
 });
 
-describe("error tests", () => {
-    test("404: responds with 404 when non-existent path when endpoint is spelt incorrectly", () => {
+describe("GET /api/articles/:article_id/comments", () => {
+    test("status 200, responds with an array of comments with the needed properties", () => {
+        return request(app)
+        .get("/api/articles/3/comments")
+        .then(({ body }) => {
+            const expected = {
+                comment_id: expect.any(Number),
+                votes: expect.any(Number),
+                created_at: expect.any(String),
+                author: expect.any(String),
+                body: expect.any(String),
+                article_id: 3
+            }
+            expect(body.comments).toBeInstanceOf(Array);
+            expect(body.comments[0]).toEqual(expected);
+        });
+    });
+    test("responds with an array where comments are sorted by date in ascending order", () => {
+        return request(app)
+        .get("/api/articles/1/comments")
+        .then(({ body }) => {
+            expect(body.comments).toBeSortedBy("created_at");
+        });
+    });
+})
+
+describe("404 error tests", () => {
+    test("status 404: responds with 404 when endpoint is spelt incorrectly", () => {
```

Comment:
You will likely have already handled a bad path test so this may have auto passed? If not then this can stay but should be in its own describe block not related to this endpoint as it doesn't pass through any of the logic for this endpoint.